### PR TITLE
Show conversion info in a more compressed format

### LINKED ISSUE #47: Show only one info line with a tooltip
Main line:
```
1 TEC = 0.00655 (?)
```
Info box:
```
Exit tribute (5%): 0.0001 xDAI
Minimum received: 0.0065 xDAI
```

### PR DESCRIPTION
**Pull request by PabloCGL**
Should close #47  
The interface now only shows how much you get for 1 unit of the supplied token, and hides the tribute etc. in the tooltip.

### CODE REVIEW

File: components/ConvertForm/ConvertForm.js [hunk 1]
```diff
@@ -154,18 +157,10 @@ function ConvertForm() {
...
-              label={`Estimated minimum received (with slippage): ${selectedOption === 0 ? inputMinWithSlippage + " "+ options[1] : inputMinWithSlippage + " " + options[0]}`}
-              description={`This tool uses a bonding curve to convert ${collateral.symbol} into ${bonded.symbol} and
-                      back at a pre-defined rate. The price is calculated by an
-                      automated market maker smart contract that defines a
-                      relationship between token price and token supply. ${selectedOption === 0 ? `You can
-                      also convert wxDAI into DAI using the xDAI bridge.` : `You can also convert ${bonded.symbol}
-                      into other tokens on Honeyswap.`}
+              label={`${selectedOption === 0 ?  `1 wxDAI = ${pricePerUnitReceived} testTEC` : `1 testTEC = ${pricePerUnitReceived} wxDAI`}  `}
+              description={`
+              ${selectedOption === 0 ? `Entry tribute (${entryTribute}%) = ${inputAmountRetained} wxDAI` : `Exit tribute (${exitTribute}%) = ${inputAmountRetained} wxDAI`}
```
Comment (sembrestels) [inline components/ConvertForm/ConvertForm.js:162]: `wxDAI` should be grabbed from `${collateral.symbol}`, and the exit tribute is specified in testTEC, so the unit should also be testTEC (`${bonded.symbol}`).

File: components/ConvertForm/useConvertInputs.js [hunk 2]
```diff
@@ -89,8 +86,11 @@ export function useConvertInputs(otherSymbol, toBonded = true) {
     const maxSlippagePct = 1 //slippage hardcoded at 1% for now, can be made changeable in the future
 
     const estReceived = amount.mul(100 - applicableTribute).div(100)
+    const amountRetained = amount.mul(applicableTribute).div(100) 
     const receivedWithSlippage = estReceived.mul(100 - maxSlippagePct).div(100)
 
+    const singleUnit = (amountSource.isZero()  ? 0 : estReceived/amountSource)
```
Comment (sembrestels) [inline components/ConvertForm/useConvertInputs.js:92]: Even if `amountSource = 0` we still should be able to fetch the price from `useBondingCurvePrice`. Could you try that, please?

File: components/ConvertForm/ConvertForm.js [hunk 3]
```diff
@@ -154,18 +157,10 @@ function ConvertForm() {
...
-              description={`This tool uses a bonding curve to convert ${collateral.symbol} into ${bonded.symbol} and
-                      back at a pre-defined rate. The price is calculated by an
-                      automated market maker smart contract that defines a
-                      relationship between token price and token supply. ${selectedOption === 0 ? `You can
-                      also convert wxDAI into DAI using the xDAI bridge.` : `You can also convert ${bonded.symbol}
-                      into other tokens on Honeyswap.`}
+              label={`${selectedOption === 0 ?  `1 wxDAI = ${pricePerUnitReceived} testTEC` : `1 testTEC = ${pricePerUnitReceived} wxDAI`}  `}
+              description={`
+              ${selectedOption === 0 ? `Entry tribute (${entryTribute}%) = ${inputAmountRetained} wxDAI` : `Exit tribute (${exitTribute}%) = ${inputAmountRetained} wxDAI`}  
+              \n Minimum received (with slippage): ${selectedOption === 0 ? inputMinWithSlippage + " "+ options[1] : inputMinWithSlippage + " " + options[0]}
```
Comment (sembrestels) [inline components/ConvertForm/ConvertForm.js:163]: `inputMinWithSlippage` is an empty string when amount is 0. It should be 0 also.

**Review by sembrestels — COMMENTED** [review]
This PR is great, thanks Nuggan! I added some comments to the places that should be changed. I also would like you change the `getTributePcts` for `useTributePcts` to continue with react hook denomination. That's almost it!

**Comment by PabloCGL** [comment]
Thanks  for the review! I made the changes and also modified the logic a bit to acually show the exit tribute in testTEC instead of wxDAI.

**Pull request merged**